### ISSUE TITLE
Deletion - 606 ONPRO, 608 VIRPRO, 616 ONATTEN, 618 VIRATTEN & Revision - 607 OFFPRO, 617 OFFATTEN

### ISSUE THREAD

**Issue by enielsen-air** [IMLS/public-libraries-survey#75]
**Name:**
Evan Nielsen

**State/Affiliation:**
American Institutes for Research (AIR)

**Description of Change:**
Delete:
606 Number of Synchronous In-Person Onsite Program Sessions (ONPRO) 
608 Number of Synchronous Virtual Program Sessions (VIRPRO) 
616 Synchronous In-Person Onsite Program Attendance (ONATTEN) 
618 Synchronous Virtual Program Attendance (VIRATTEN) 

Revise to be subsets of TOTPRO and TOTATTEN, respectively:
607 Number of Synchronous In-Person Offsite Program Sessions (OFFPRO) 
617 Synchronous In-Person Offsite Program Attendance (OFFATTEN) 

**Justification:**
Implementing the ON/OFF/VIR programming items as a parallel set of items parallel to the age categories that sum to the total has been challenging (to put it mildly). In the aftermath of the COVID-19 pandemic, virtual programs have become much less important than they were then these items were proposed four years ago.

In a preliminary look at the VERY fresh (not yet processed) FY23 data, about 64% of AEs overall reported **zero** virtual programs and attendance, and another 11% of AEs reported fewer than 10 virtual programs. There are 18 states with at least three quarters of AEs reporting zero virtual programs and attendance.

From what I've heard from SDCs, the most important metric of these three to capture is the **off-site** programs. So it would be much easier to report and much easier to capture off-site programs and attendance as a subset of total programs and attendance. (Of course, states would be free to capture off-site as a subset of each age category if they wanted to, but that detail would not be captured in the PLS.)

**Methodological Issues:**
This proposal would resolve several current methodological issues, including the impending imputation of these items. It is unclear whether OFFPRO and OFFATTEN would need to be revised. And minor revisions to TOTPRO and TOTATTEN may be necessary to remove references to how ON/OFF/VIR items sum to the totals.

**Comment by KristenCooke** [comment]
I am overjoyed to see this proposal. The measurement of onsite and offsite is the most informative measurement for libraries to make strategic operating decisions and to communicate the level of service the library provides to the community beyond the library's four corners. The continued measurement of virtual program offerings when they are no longer service-crucial paints a picture of a library underperforming in a category rather than a picture of a library delivering need-informed services.

**Comment by megdepriest** [comment]
I agree with Kristen!

**Comment by kkrumsee** [comment]
I'm very glad to see this proposal as well. More than any other area of the survey, these questions have been causing the most confusion for my libraries

**Comment by nrgieske** [comment]
Yes please!!! The programming section desperately needs to be simplified. In Kentucky, in general, libraries are not doing 100% virtual programming anymore. On-site versus off-site is a better measure now than in-person versus virtual or hybrid.
